src/components/ImportJobs/ImportJobs.js

```diff
@@ -133,8 +146,8 @@ class ImportJobs extends Component {
       >
         {openFileUploadDialogWindow => (
           <InvalidFilesModal
-            isModalOpen={isModalOpen}
-            onConfirmModal={this.hideModal}
+            isModalOpen={filesExtensionsModalOpen}
```

Comment:
Let's rename this prop to be just `isOpen`. The name of the component already makes it clear that the prop relates to the `modal`.